blocks/api/raw-handling/test/integration/index.js

```diff
@@ -8,7 +8,7 @@ import path from 'path';
 /**
  * Internal dependencies
  */
-import { registerCoreBlocks } from '../../../../library';
+import { registerCoreBlocks } from '../../../../../core-blocks';
```

Comment:
Should we move this test to `core-block`? It seems to be very dependent on them.

Reply:
yes, maybe I'll defer to @iseulde and probably leave this to a separate task?

Also, maybe we could improve the core block tests we have now. They are all pretty equivalent relying on `blockEditRender` and test a snapshot. I think we should improve them one by one and just call the `edit`, `save`, `transforms` etc... without any mocking unless we can't do that for `edit` for a reason I'm missing?